node/mock-test.ts

```diff
@@ -167,21 +167,17 @@ export class MockTestRunner {
     // Returns a path to node.exe with the correct version for this task (based on if its node10 or node)
     private async getNodePath(nodeVersion?: number): Promise<string> {
         const version: number = nodeVersion || this.getNodeVersion();
-
-        let downloadVersion: string;
-        switch (version) {
-            case 6:
-                downloadVersion = 'v6.17.1';
-                break;
-            case 10:
-                downloadVersion = 'v10.21.0';
-                break;
-            case 16:
-                downloadVersion = 'v16.13.0';
-                break;
-            default:
-                throw new Error('Invalid node version, must be 6, 10, or 16 (received ' + version + ')');
-        }
+        const versions = {
+            20: 'v20.11.1',
```

Comment:
Is this correct version of Node 20? Shouldn't we use the latest (20.13.1)? @kirill-ivlev

Reply:
We should, when I checked last time the agent used 20.11.1. Fixed, thanks!